spec/compiler/crystal/tools/init_spec.cr

```diff
@@ -1,149 +1,175 @@
-require "spec"
-require "yaml"
-require "ini"
 require "compiler/crystal/config"
 require "compiler/crystal/tools/init"
+require "file_utils"
+require "ini"
+require "spec"
+require "yaml"
+
+BIN_CRYSTAL = File.expand_path("bin/crystal")
```

Comment:
This assumes `Dir.current` is Crystal's root directory, which might not be the case. I recommend adjust it so `bin/crystal` can be calculated relatively to this file/directory location instead.

```
$ which ccr
/home/luis/bin/ccr

$ file `which ccr`
/home/luis/bin/ccr: symbolic link to /home/luis/code/crystal-lang/crystal/bin/crystal

$ cd spec/compiler/crystal/tools

$ ccr eval 'p Dir.current, File.expand_path("bin/crystal")'
Using compiled compiler at `.build/crystal'
"/mnt/c/Users/Luis/Code/crystal-lang/crystal/spec/compiler/crystal/tools"
"/mnt/c/Users/Luis/Code/crystal-lang/crystal/spec/compiler/crystal/tools/bin/crystal"
```

Reply:
True, but it didn't work before too

Reply:
Regarding specs, it should be reasonable to assume they are run from the project root folder. I don't think it is worth fixing this unless there is a specific issue why this wouldn't work.

Reply:
@luislavena ah ok I see what you mean, I fixed it!